### PR TITLE
Functions to modify points labels

### PR DESCRIPTION
**Pull request by f-fl0**
This PR adds a couple of function to modify the points labels based on a specified label range or a list of labels to keep unchanged. Here is a video of these features in action:

https://user-images.githubusercontent.com/66578286/176340263-c29552bb-13cf-4bb1-ae82-e227deb7ef6a.mp4

### CODE REVIEW

**Review by at-wat — COMMENTED** [review]
I think it's better to avoid expanding default menu items too much.
What do you think?

**Comment by f-fl0** [comment]
> I think it's better to avoid expanding default menu items too much.

You are probably right. I can try to put those new options to another menu (e.g. `advanced` or something) or take them out completely. The added functions can be triggered via the command panel anyway.

**Review by at-wat — COMMENTED** [review]
Could you add commands to README.md?

File: README.md [hunk 1]
```diff
@@ -124,6 +124,9 @@ segmentation\_param `D` `R`   | セグメンテーション時の分離距離を
 fit\_inserting `AXIS`...      | 貼り付け中の点群を既存の点群に位置合わせ [\*2](#footnoteKey2) (位置合わせを行う軸 `AXIS` をスペース区切りで複数指定 [\*3](#footnoteKey3))
 label_segmentation\_param         | ラベルを元にしてのセグメンテーション時の範囲と隣接する点群の最大距離を表示 [\*1](#footnoteKey1)
 label_segmentation\_param `D` `R` | ラベルを元にしてのセグメンテーション時の隣接する点群の最大距離を `D` \[メートル\]、範囲を `R` \[メートル\]に設定
+render_label_range\_param `Min` `Max` | `Min` - `Max`の範囲内のラベルだけを色をつける
+relabel\_param `Min` `Max` `New` | `Min` - `Max`の範囲内のラベルを`New`値に設定
+unlabel\_param `label1` `label2` `...` | `label1, label2, ...`以外のラベル`0`に設定
```
Comment (at-wat) [inline README.md:129]:
```suggestion
render_label_range\_param `Min` `Max` | `Min` - `Max`の範囲内のラベルのみに色をつけて表示
relabel\_param `Min` `Max` `New` | `Min` - `Max`の範囲内のラベルを`New`値に設定
unlabel\_param `label1` `label2` `...` | `label1, label2, ...`以外のラベルを`0`に設定
```

**Review by at-wat — APPROVED** [review]
LGTM

**Pull request merged**